install/Docker/dockerfiles/scripts/build_cs.sh

```diff
@@ -1,4 +1,4 @@
-#(C) Copyright [2022] American Megatrends International LLC
+#(C) Copyright [2020] American Megatrends International LLC
```

Comment:
do not revert license information that is already published

Reply:
updated in new commit